src/components/KnockFeedProvider/KnockFeedProvider.tsx

```diff
@@ -44,30 +46,40 @@ export const KnockFeedProvider: React.FC<KnockFeedProviderProps> = ({
   colorMode = "light",
   rootless,
 }) => {
+  const knockInstance = React.useRef<Knock | null>(null);
```

Comment:
Would it make sense to add comments on what's going on? (React 18 strict mode on dev)